scripts/server-notifier.coffee

```diff
@@ -24,7 +24,8 @@ module.exports = (robot) ->
 
     try
       #data = querystring.stringify(req.body)
-      data = JSON.stringify(req.body)
+      #data = JSON.stringify(req.body)
+      data = "#{req.body.title} - #{new Date()}\n#{req.body.data}\n"
```

Comment:
new Date()
Do you think the time between our server and hubot server (heroku) is in sync?